iOS/WAStickersThirdParty/StickerPackInfoViewController.swift

```diff
@@ -38,18 +39,17 @@ class StickerPackInfoViewController: UITableViewController {
 
         if stickerPack.publisherWebsite != nil {
             websiteIndexes[.publisher] = index
-            index += 1
         }
 
         if stickerPack.privacyPolicyWebsite != nil {
             websiteIndexes[.privacyPolicy] = index
-            index += 1
         }
 
         if stickerPack.licenseAgreementWebsite != nil {
             websiteIndexes[.licenseAgreement] = index
-            index += 1
         }
+
+        index += 1
```

Comment:
I think that increasing the index on each if statement (expect maybe the last one) is the correct way to go, just like before. With the new change, you can actually prevent one or more of the website cells from showing.

Reply:
✅ Done

Thanks for your attention @amrit-1901 I updated this pull-request please take a look